### DISCUSSION TITLE
Push demo firmware edits

### DISCUSSION THREAD

**Discussion by cli48** [bitcraze/discussions#1284] [Q&A]
Hello, I am trying to use the AI deck, multiranger and flow deck v2 together on a single crazy fly. My thought process is, the push demo is essentially done for me, now I need to configure the wifi in the configs. I cannot use the 'make menuconfig' command, I attempted to edit the app-config file in the push demo by adding wifi but im not sure how to target which binary i want to flash when i use 'make cload'. I would greatly appreciate assistance.

**Comment by gemenerik** [comment]
Hi Calvin. I don't completely understand your question. Are you suggesting you get an error when trying to `make menuconfig`, or do you deliberately want to configure the WiFi without it?

Using menuconfig generates the `build/.config` file, with the following defines.
```
# Expansion deck configuration
CONFIG_DECK_AI=y
# CONFIG_DECK_AI_WIFI_NO_SETUP is not set
CONFIG_DECK_AI_WIFI_SETUP_AP=y
# CONFIG_DECK_AI_WIFI_SETUP_STA is not set

#
# Credentials for access-point
#
CONFIG_DECK_AI_SSID="calvinsaideck"
CONFIG_DECK_AI_PASSWORD="123456789"
```

I would recommend just using the menuconfig, but otherwise you can overwrite these defines from somewhere else in the firmware.

**Reply by cli48** [reply]
Hello, 
I seek to modify the push demo firmware and add a WiFi access point to also utilize the AI deck 1.1. When in the directory for the push demo, the 'make menuconfig' is not accessible. I will try using what you've given to test on the push demo.

**Reply by gemenerik** [reply]
You can make a configuration with `make menuconfig` in the root firmware directory and copy the contents of `crazyflie-firmware/build/.config` into your `app-config` file (the one with `CONFIG_APP_ENABLE=y` etc, keep those in by the way).

**Reply by gemenerik** [reply]
Files and folders starting with a dot are hidden by default. `ls -a` in your terminal or "Show Hidden Files" / `ctrl+h` in Nautilus.
